torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -164,7 +164,16 @@ void CudaPrinter::visit(const Load* v) {
     os() << "__half2float(" << *v->base_handle() << "[" << *v->flat_index()
          << "])";
   } else {
-    os() << "__ldg(" << *v->base_handle() << " + " << *v->flat_index() << ")";
+    // Detects whether the load target is also a store target.
+    // TODO: this is currently too wide. It detects whether a store-target exsit
```

Comment:
nit: "exists"

in general, should we plan to add the concept of memory const-ness to the IR?  weights could be handled differently etc